gapipy/models/base.py

```diff
@@ -126,7 +126,7 @@ def _set_resource_field(self, field, value):
             for v in value:
                 stub.append(self._model_cls(field)(v, stub=True))
         elif value:
-            stub = self._model_cls(field)(value, stub=True)
+            stub = self._model_cls(field)(value, self._client, stub=True)
         setattr(self, field, stub)
 
     def _set_resource_collection_field(self, field, value):
```

Comment:
In cases where the `Client` is returning us some sort of `Resource` it can set a reference to itself on `foo._client` before returning it.

In cases where we instantiate a `Resource` ourselves we're probably trying to create and save a new thing, no? This may be a not-great idea, but is it super-disruptive to require the user to supply `Client` in order to save that `Resource`? [It looks like `Resource.create` actually already takes a `client` arg](https://github.com/gadventures/gapipy/blob/6d19616cf14d435a76067a94879a69ea63ef4a6a/gapipy/resources/base.py#L34) and I guess `save`, `_create`, and `_update`, would have to change to receive a `client` (but could stash it on `self._client` for later use)